sources/Interop/WinTrust/um/wintrust/CRYPT_PROVIDER_SGNR.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/WinTrust.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct CRYPT_PROVIDER_SGNR
+    {
+        [NativeTypeName("DWORD")]
+        public uint cbStruct;
+
+        [NativeTypeName("FILETIME")]
+        public _FILETIME sftVerifyAsOf;
```

Comment:
Needs remapping, should be already defined in `TerraFX.Interop.Windows`

Reply:
Fixed.